src/components/side_panel/conditional_formatting/cf_preview/cf_preview.ts

```diff
@@ -138,8 +137,7 @@ export class ConditionalFormatPreview extends Component<Props, SpreadsheetChildE
     return this.props.conditionalFormat.ranges.map((range) => ({
       sheetId,
       zone: this.env.model.getters.getRangeFromSheetXC(sheetId, range).zone,
-      color: SECONDARY_COLOR,
-      noFill: true,
+      fillAlpha: 0.025,
```

Comment:
It's weird to have the alpha specified but not the color. You have to know the default color for this alpha value to make sense.
I would have kept the color required and specified here.